src/App/App.js

```diff
@@ -1,31 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Home } from '../Home/Home';
+import { Header } from '../Header/Header';
 import BookDetails from '../BookDetails/BookDetails';
 import { Switch, Route } from 'react-router-dom';
 import Login from '../Login/Login';
 
-const App = ({currentUser}) => {
+const App = () => {
+
+  const [currentUser, setCurrentUser] = useState({});
+
```

Comment:
This looks great! Thanks for cleaning up the routing. I think passing in user from app makes since because of where we later need to pass it down. Great Work!